test/bigtest/helpers/translationsProperties.js

```diff
@@ -17,6 +17,7 @@ export const translationsProperties = [
       hideSearchPane: 'hideSearchPane',
       showSearchPane: 'showSearchPane',
       numberOfFilters: 'numberOfFilters',
+      searchResultsCountHeader: '{count} {count, plural, one {record found} other {records found}}',
```

Comment:
As far as I understand the idea, this string is used from stripes-smart-components. Should this translation be hardcoded here If translations from stripes components and other external modules can be exported for testing purposes (For example, several lines above)?

Reply:
I thought about that, but I don't want to add a dependency on stripes-smart-components, just for the sake of translations. This translation is not likely to change in my opinion.

Reply:
Okey, sounds reasonable. Maybe, this is a good topic to discuss if we should use translations from other modules at all or provide a mechanism to deal with it to prevent the problem like this in the future.

Reply:
As a possible resolution for this case, I was thinking about introducing our own translations with this key in order to not depend on stripes-smart-components. What do you think?

Reply:
I think, it sounds good, because now we have to dive into stripes-smart-components (copy translations) to make this check and some unnecessary (message update, translation key update) updates will affect us. The downside is that we will have to manage the string update. But this is a separated module, so it's okay.